mas_perception_libs/README.md

```diff
@@ -11,6 +11,37 @@ Contains shared perception defintions in C++.
 Contains shared perception definitions in Python.
```

Comment:
Your documentation is generally great, so nicely done! I do have a couple of improvement suggestions though:
* since `perception_libs` is a relatively large piece of software, it would be very helpful to add a high-level diagram in the README introduction (or a set of diagrams - one per section) that gives an overview of the whole library and thus helps anyone who's trying to use it
* I think the introduction should also explain the rationale behind having both a C++ library and a Python interface (i.e. what's the purpose of both); short descriptions of the two should also be included in the sections dedicated to them

Reply:
added image detection diagram and explanation for C++ Python interface